spec/integration/comment_integration_spec.rb

```diff
@@ -191,21 +201,21 @@ module DiasporaFederation
       end
 
       it "relays legacy signatures and xml" do
```

Comment:
You're right, the focus is on "order" and not "signature", so I changed that. I keeped "legacy", so it's "legacy order" and "new order" for now (instead of "random order 1 or 2"). I also did some other tweaks to make `let`s and test descriptions better understandable. I think that's OK for now, since these tests will change anyway, when we drop legacy xml format parsing. And I think we can drop the different order tests then, because the "new data" tests also includes that somehow.